### DISCUSSION TITLE
[SOLVED] SONOFF TRVZB thermal head buzzes every Monday at 12:00

### DISCUSSION THREAD

**Discussion by vukitoso** [Koenkk/zigbee2mqtt#25224] [Devices]
Hi all.
Has anyone observed such an interesting moment that the SONOFF TRVZB thermal head, every Monday, at approximately 12:00 - 12:02, buzzes for about 10 seconds and sends a message about its status.
It takes 3-4 seconds to open or close, but here it takes as much as 10 seconds.
Both buzz, sometimes almost simultaneously, sometimes with a time difference.
What is this buzz on Mondays at 12? This is the third Monday in a row that it has worked.
Both are connected via zigbee2mqtt.

SONOFF TRVZB
Firmware build date: 20240823
Firmware version: 1.2.1
Zigbee2MQTT: 1.42.0 commit: 861cba6

**Comment by krzysztofczernek** [comment]
It's the anti-calcification function – see the manual.

**Reply by vukitoso** [reply]
> It's the anti-calcification function – see the manual.

I found the instructions, read them, but didn’t find anything about the “anti-calcification function”.
Here is a [link to the PDF instructions](https://sonoff.tech/wp-content/uploads/2023/09/%E8%AF%B4%E6%98%8E%E4%B9%A6-TRVZB-V1.1.pdf)
Can you point your finger at where this is described?

**Reply by krzysztofczernek** [reply]
Page 9, in the middle

**Reply by vukitoso** [reply]
Thx.
Unfortunately, this is not in my paper instructions.
